sdk/build.gradle

```diff
@@ -70,7 +70,7 @@ tasks.register('dokkaJavadocJar', Jar.class) {
 // Maven Publishing
 
 mavenPublishing {
-    publishToMavenCentral(SonatypeHost.S01)
+    publishToMavenCentral(SonatypeHost.S01, true)
```

Comment:
This lets the plugin know that it's configured as part of `automaticRelease`. This means that if the version being published ends with `-SNAPSHOT`, the artifacts will be published to Sonatype's snapshot repository instead.